is_core/tests/crawler.py

```diff
@@ -121,7 +112,7 @@ def pre_request(url, referer, headers):
 
         def post_response(url, referer, resp, exception):
             tested_urls.append(url)
-            assert_true(exception is None or isinstance(exception, HTMLParseError),
+            assert_true(exception is None or isinstance(exception),
```

Comment:
co to isinstance dělá, když tam neni classa?

Reply:
:D no asi spadne